src/get.ts

```diff
@@ -11,7 +11,7 @@ chel get https://url.com mygreatlongkey > file.png
 import { flags, writeAll } from './deps.ts'
 import { exit, getBackend, isURL, readRemoteData } from './utils.ts'
 
-export async function get (args: string[]): Promise<void> {
```

Comment:
All of these "space around functions" changes need to be reverted. They seem to violate standard too, check `group-income`, doing this results in:

```
225:20  error  Missing space before function parentheses  space-before-function-paren
```